examples/siamese_network/siamese_network.py

```diff
@@ -0,0 +1,325 @@
+from __future__ import print_function
```

Comment:
Do we really need this ?

Reply:
Now that I think of it, since python version used is > than 3.0 hence it will not be needed